aggregator/aggregator.go

```diff
@@ -44,6 +44,11 @@ var AggregatorCommand = &cli.Command{
 			Value:   "15s",
 			Usage:   "Time (in seconds) to wait between each aggregation cycle",
 		},
+		&cli.IntFlag{
+			Name:  "threshold-percentage",
+			Value: 85,
+			Usage: "If the number of eligible nodes goes below the threshold, npd will stop marking nodes as ineligible",
```

Comment:
This is something that can wait until later but maybe we should make this a `AFTER the number of eligible nodes goes below the threshold`. The reason being someone might spin this up against a nomad cluster with 3 nodes and conclude that this doesn't work after breaking one of the nodes. This is a UX suggestion - not a functional suggestion that's relevant for it to work at Roblox.